service/src/main/resources/api/service_openapi.yaml

```diff
@@ -607,22 +607,22 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
-  /api/workspaces/v1/{workspaceId}/resources/referenced/gcp/bucket/files:
```

Comment:
This is also a breaking API change, though I see the CLI PR for GCS objects is still pending and the UI doesn't use this endpoint yet, so it's probably ok.

Reply:
yeah i'm planning to wait till this one goes in to update the CLI relevant PR.